### ISSUE TITLE
[Impl] message validation

### ISSUE THREAD

**Issue by yudetamago** [LayerXcom/cbc-casper-simulator#1]

**Comment by yudetamago** [comment]
See:
* https://github.com/LayerXcom/cbc-beacon-chain/issues/1#issuecomment-478194477
* https://github.com/LayerXcom/cbc-casper-simulator/blob/master/cbc_casper_simulator/message_validator.py

**Comment by nrryuya** [comment]
Same thought with https://github.com/LayerXcom/cbc-casper-simulator/issues/4#issuecomment-480589390

**Comment by yudetamago** [comment]
Makes sense, complete validation is not necessary, but some of simplified validations (like `justification_is_justified` as described above) are useful for debugging.
Specifically, We might use it as just an `assertion` for experiments like: https://github.com/LayerXcom/cbc-casper-simulator/blob/de186bdfb1d1eafa70dcd78661b02459f6098c7a/cbc_casper_simulator/simulator/broadcast_and_receive_simulator.py#L35-L36

**Comment by yudetamago** [comment]
About `justification_is_justified`, it is required to check whether the message is to be "pending" or not.
